### PR TITLE
Feat/Profile image upload

### PR DESCRIPTION
**Pull request by jphinning**
- Added the possibility to upload profile images to the user
image

- Profile images will now be appearing on any job detail, offer, or talent page. Example below 👇🏻 
image

### CODE REVIEW

**Review by milkavladislav — APPROVED** [review]
Don't have any comments

File: src/components/UI/drawers/JobOfferDrawer/JobOfferDrawer.tsx [hunk 1]
```diff
@@ -120,7 +119,7 @@ function JobOfferDrawer({
             <ProfileBox showBorder={!(status === OfferStatus.PENDING)}>
               <CustomText strong>{t("SearchWork.clientInfo")}</CustomText>
               <Space direction="vertical" align="center">
-                <img height="50px" alt="logo" src={logo} />
+                <Avatar size={70} alt="logo" src={jobPost?.user?.avatar?.url} />
```
Comment (lyudad) [inline src/components/UI/drawers/JobOfferDrawer/JobOfferDrawer.tsx]: I see that you use 70several times
maybe it make sense move it to AVATAR_SIZE - constant?

Comment (jphinning) [reply src/components/UI/drawers/JobOfferDrawer/JobOfferDrawer.tsx]: Good idea, added and applied avatar size consts to all cards that were using it

**Comment by lyudad** [comment]
Снимок экрана 2022-10-10 в 21 26 23

**Pull request merged**